packages/create-plugin/templates/app/provisioning/plugins/apps.yaml

```
@@ -1,5 +1,11 @@
 apiVersion: 1
 
 apps:
-  - type: '{{ normalize_id pluginName orgName 'app' }}'
+  - type: '{{ normalize_id pluginName orgName 'app' }}')
```

There is an accidental `)` at the end of the line.
```suggestion
  - type: '{{ normalize_id pluginName orgName 'app' }}'
```